assets/js/admin/ppec-upgrade-notice.js

```diff
@@ -0,0 +1,43 @@
+;(function ( $, window, document ) {
+	'use strict';
+
+	// Check whether PayPal Payments is installed.
+	let is_paypal_payments_installed = false,
+		is_paypal_payments_active = false;
+
+	const targetElement = $( 'tr[data-slug="woocommerce-paypal-payments"]' );
+	if ( targetElement.length ) {
+		is_paypal_payments_installed = true;
+
+		if ( targetElement.hasClass( 'active' ) ) {
+			is_paypal_payments_active = true;
+		}
+	}
+
+	// Hide notice/buttons conditionally.
+	if ( is_paypal_payments_installed && is_paypal_payments_active ) {
```

Comment:
Instead of trying to hide things when the JS is loaded, could we maybe hide the notice and buttons _by default_ using the CSS file (or even a `style: display: none;` inline attribute if necessary) and only show the relevant buttons if needed? Otherwise, even though it's not easy to see, the screen would flicker while the initial buttons are shown/hidden.

This is not critical. Feel free to skip this request completely if you're busy with other stuff.

Reply:
Made the entire buttons area hidden by default. Gets displayed when JS initializes.